src/sentry/static/sentry/app/components/modals/sentryAppDetailsModal.tsx

```diff
@@ -88,7 +88,7 @@ export default class SentryAppDetailsModal extends AsyncComponent<Props, State>
     return [['featureData', `/sentry-apps/${sentryApp.slug}/features/`]];
   }
 
-  featureTags(features: IntegrationFeature[]) {
+  featureTags(features: Pick<IntegrationFeature, 'featureGate'>[]) {
```

Comment:
@EvanPurkhiser this is bit wonky for me because the type of `features` will have more than that. Though I know it works because we only use `featureGate`. We should have a comment explaining this discrepancy.

Reply:
Yeah I can leave a comment if we really want, but tbh i think this is kinda fine.

Reply:
lol fine, this component will be gone soon anyways